### PR TITLE
Add vsphere-cpi 1.22.6 1.21.3 1.20.1

### PR DESCRIPTION
**Pull request by lubronzhan**
## What this PR does / why we need it

This PR bumps CPI 1.22.6, 1.21.3, 1.20.1, which have the fix for IP disappeared for node after node is restarted

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Review by HanFa — DISMISSED** [review]
LGTM

**Review by seemiller — CHANGES_REQUESTED** [review]
I left 3 suggestions to update the image SHA in the package.yaml files. Are you using imgpkg to push the packages to Harbor? If so, I would recommend updating to a newer version. All of the new bundles that were pushed for this PR only have 1 tag. If that tag is ever removed, Harbor will garbage collect and delete that image. Newer versions of imgpkg will add the SHA as a tag to prevent this.

File: addons/packages/vsphere-cpi/1.20.1/package.yaml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  refName: vsphere-cpi.community.tanzu.vmware.com
+  version: 1.20.1
+  releaseNotes: vsphere-cpi 1.20.1 https://github.com/kubernetes/cloud-provider-vsphere
+  licenses:
+  - Apache 2.0
+  template:
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/vsphere-cpi@sha256:3764207049a4b0b7d8e18073e6a105ff5edea87e397292c2a893ece756055103
```
[the remark this answers is not in the document]
Comment (HanFa) [reply addons/packages/vsphere-cpi/1.20.1/package.yaml:15]: the SHA I got is `be17e733f2ef2a48184d6fd6aed21876ffd0f9149c15455ec3368cfc4227d2c2`

File: addons/packages/vsphere-cpi/1.22.6/package.yaml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  refName: vsphere-cpi.community.tanzu.vmware.com
+  version: 1.22.6
+  releaseNotes: vsphere-cpi 1.22.6 https://github.com/kubernetes/cloud-provider-vsphere
+  licenses:
+  - Apache 2.0
+  template:
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/vsphere-cpi@sha256:a132e8b4eef8c5df5c816b9a8fa41a2e418f1de7fde5b22b32e1d9896ed74d01
```
[the remark this answers is not in the document]
Comment (HanFa) [reply addons/packages/vsphere-cpi/1.22.6/package.yaml:15]: SHA I got is the same `3201da54c4a87672949f038697da883575f86755fcc32137409ae85c38d720d7`

File: addons/packages/vsphere-cpi/1.21.3/package.yaml [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+  refName: vsphere-cpi.community.tanzu.vmware.com
+  version: 1.21.3
+  releaseNotes: vsphere-cpi 1.21.3 https://github.com/kubernetes/cloud-provider-vsphere
+  licenses:
+  - Apache 2.0
+  template:
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/vsphere-cpi@sha256:8b9442c8f2159fbd833da7dcc1a0f6a727340e23c4b5ca82d8287a96a5c8ffa5
```
[the remark this answers is not in the document]
Comment (HanFa) [reply addons/packages/vsphere-cpi/1.21.3/package.yaml:15]: SHA is the same `76738506b9d31ed56dca180a8e9585932a8fd96d8e6d97ffdaae61a9a3ea8fc1`

**Review by seemiller — APPROVED** [review]

**Pull request merged**